docs/05_operations/13_project-structure/02_settings-plugin.mdx

```diff
@@ -0,0 +1,202 @@
+
+Differences between the two:
+- When the project type is PBC, by default, the project's dictionary cache sub-modules have publishing tasks disabled. If you want to enable publishing then disable the `genesisPublicationDisabler` plugin (see [here](#plugins) for details).
+- When the project type is APPLICATION, a site-specific distribution is created in addition to the project's one.
+
+```kotlin
+genesis {
+    projectType = APPLICATION
+}
+```
+
+### Dependencies
+
+Here you should declare dependencies on other Genesis products, then the Settings plugin can take care of where they should be added automatically. 
+The Settings plugin will do the following with these dependency declarations:
+- Add a dependency on the product's config to your project's dictionary cache for code generation
+- Capture runtime details of the product's processes (e.g. for running processes with the IntelliJ plugin or the Deploy plugin)
+
+The syntax is similar to Gradle dependencies but not exactly the same. Specifically, you specify the group id, product name and version: `"{groupId}:{productName}:{version}"`
+
+Here is an example of adding a dependency on Auth and Genesis Notify:
+
+```kotlin
+genesis {
+    dependencies {
+        dependency("global.genesis:auth:${extra.properties["authVersion"]}")
+        dependency("global.genesis:genesis-notify:${extra.properties["notifyVersion"]}")
+    }
+}
+```
+
+### Plugins
+
+Other Gradle plugins that are necessary for your project are automatically added by the Settings plugin. Below you can see the plugins that are added for both Legacy and Simplified structures. These plugins can also be disabled if needed.
+
+<Tabs defaultValue="Simplified Structure" values={[{ label: 'Simplified Structure', value: 'Simplified Structure', },{ label: 'Legacy Structure', value: 'Legacy Structure'}]}>
+<TabItem value="Simplified Structure">
+
+| Plugin Name | Plugin ID | Description | Enabled by default |
+| --- | --- | --- | --- |
+| Kotlin | org.jetbrains.kotlin.jvm | Kotlin Gradle plugin | Yes |
+| Genesis Build | global.genesis.build | Core Genesis plugin to enable build tasks, including code generation | Yes |
+| Genesis Distribution | global.genesis.distribution | Creates the project's server distribution zip file. At the moment it does this based on default configuration within the plugin. If you want to customise your distribution then you will need to disable this plugin and create your own configuration using the Gradle distribution plugin. In the future we will support customisation from this plugin directly. | Yes |
+| Genesis Publication Disabler | global.genesis.publication-disabler | Used to disable dictionary-cache sub-module publishing tasks if the project is of type PBC | Yes |
+| Genesis Exec | global.genesis.exec | Creates Gradle tasks used by the Genesis IntelliJ plugin | Yes |
```

Comment:
I am not sure whether to mention about exec plugin yet! as all its functionality is exposed through intellij plugin